python/cuml/cuml/fil/fil.py

```diff
@@ -19,6 +19,8 @@
 from cuml.experimental.fil.fil import (
     ForestInference as ExperimentalForestInference,
 )
+from cuml.experimental.fil.fil import get_fil_device_type  # noqa
+from cuml.experimental.fil.fil import set_fil_device_type  # noqa
```

Comment:
I'd recommend defining an `__all__` module variable instead of silencing the linter with `# noqa`.